### PR TITLE
fix(modal): fix initial focus of modal focus trap

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: addon/templates/components/uk-modal.hbs [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
-      {{#if this.btnClose}}<button class="uk-modal-close-default" type="button" uk-close></button>{{/if}}
+    <div
+      class="uk-modal-dialog {{@dialogClass}}"
+      role="dialog"
+      aria-modal="true"
+      aria-labelledby={{this.modalHeaderId}}
+      tabindex="-1"
+      {{focus-trap
+        isActive=this.visible
+        shouldSelfFocus=true
```
Comment (czosel) [inline addon/templates/components/uk-modal.hbs]: This is the actual fix?

File: addon/templates/components/uk-modal.hbs [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+        focusTrapOptions=(hash
+          escapeDeactivates=true
+          clickOutsideDeactivates=true
+          returnFocusOnDeactivate=true
```
Comment (czosel) [inline addon/templates/components/uk-modal.hbs]: or this? ^^

Comment (anehx) [reply addon/templates/components/uk-modal.hbs]: Both

**Pull request merged**